specificatie/persoon.yaml

```diff
@@ -104,7 +104,11 @@ components:
           $ref: '#/components/schemas/GbaGeheimhoudingPersoonsgegevens'
         geslacht:
           $ref: 'common.yaml#/components/schemas/Waardetabel'
-        inOnderzoek:
+        persoonInOnderzoek:
+          $ref: 'gba-inonderzoek.yaml#/components/schemas/GbaInOnderzoek'
+        gemeenteInOnderzoek:
```

Comment:
Ik denk dat het onterecht was dat in https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/1075#issuecomment-1134594109 stond dat er ook een gemeenteInOnderzoek nodig is. Die staat immers al in verblijfplaats. Dus het verblijfplaats.inOnderzoek kan betrekking hebben op de gemeente, op immigratie op adressering en op verblijfplaats. Ik denk dat het niet nodig is om diezelfde inOnderzoek uit categorie 7 ook nog in gemeenteInOnderzoek te zetten

Reply:
Ik heb gemeenteInOnderzoek verwijderd